gems/activerecord/6.0/activerecord.rbs

```diff
@@ -39,6 +39,7 @@ module ActiveRecord
     extend ::ActiveSupport::Callbacks::ClassMethods
 
     def self.abstract_class=: (bool) -> void
+    def self.default_timezone=: (*untyped) -> untyped
```

Comment:
```suggestion
    def self.default_timezone=: (untyped) -> untyped
```


This method doesn't receive a variadic argument but it receives just one argument.

Reply:
Yea thank you!